packages/dataviews/src/index.ts

```diff
@@ -1,5 +1,5 @@
-export { default as DataViews } from './dataviews';
+export { default as DataViews } from './components/dataviews';
```

Comment:
What advantage do you see with adding an extra folder layer with `components` folder? Additionally every sub folder here has the `dataviews` prefix. Do you think we need this  since we're in dataviews package?

Reply:
> What advantages do you see with adding an extra folder layer with components folder? 

- We have another folder called "layouts" for the default layouts
- We'll have more folders later (field types)
- Components might not be the perfect folder name here but I like that all the components that make up the dataviews and data forms are now in a folder to avoid being mixed with "layouts", "field types"...

> Do you think we need this since we're in dataviews package?

Yes, because we have. `DataViews` and `DataForm`. DataForm can also potentially be composed later like the DataViews.

Reply:
>Components might not be the perfect folder name here but I like that all the components that make up the dataviews and data forms are now in a folder to avoid being mixed with "layouts", "field types"

I'm fine with it, although my preference for this package is to organise without the components. I think it makes it easier find files related per 'function'. What I mean by this, is have the folder `filters` for example and add there all related code (filters, global search, etc..).

Reply:
Other than filters, the other components don't really have sub components, so a folder doesn't really make sense IMO.